build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,7 +7,9 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.0.2";
-        public static readonly string JitVersion = "1.0.2";
+        // TODO: Update these for consuming servicing version of CoreCLR packages
+        //       Also, update CoreCLR package referenced at /Users/gkhanna/Github/gkhanna79/core-setup/pkg/projects/Microsoft.NETCore.App/project.json
```

Comment:
@ellismg My plan was to not merge this specific PR but update it once CoreCLR side of the changes have been published so that I refer to the 1.0.5 versions.